events/elections/2020/README.md

```diff
@@ -110,25 +125,27 @@ enforced as spelled out in the [Steering Committee Election Charter].
 You will be ranking your choices of the candidates with an option for
 "no opinion". In the event of a tie, a coin will be flipped.
 
-The election will open for voting the morning September 28, 2019 via email and
-end two weeks after on October 2, 2019 at 00:00am UTC. You will receive an email
+The election will open for voting on September 14th and 15th via email and
```

Comment:
Why also the 15th? :thinking:

Reply:
probably bc of timezones but if that's the case I would explicitly state that so folks aren't confused

Reply:
Because we've been warned that emailing of ballots by CIVS can take many hours, espectially if you take 2nd rounds to alternate email addresses into account.

Reply:
Agreed that wording is confusing, though, so changing it.